PrebidMobile/ConfigurationAndTargeting/Targeting.swift

```diff
@@ -255,6 +257,45 @@ public class Targeting: NSObject {
         return transformedUserIdArray.isEmpty ? nil : transformedUserIdArray
     }
     
+    // MARK: - SharedId
+    
+    public var sharedIdEnabled: Bool = false
```

Comment:
Please document the public properties and methods according to this guide: https://www.swift.org/documentation/docc/writing-symbol-documentation-in-your-source-files